notebooks/week3/02.model_serving.py

```
@@ -0,0 +1,185 @@
+        pass
+    else:
+        raise e
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ## Call the endpoint
+
+# COMMAND ----------
+
+token = dbutils.notebook.entry_point.getDbutils().notebook().getContext().apiToken().get()
```

Ruff

Undefined name `dbutils`

(F821)